relay.go

```diff
@@ -64,43 +62,3 @@ func BuildRelayRequest(
 		Payload: requestBz,
 	}, nil
 }
-
-// ValidateRelayResponse validates the RelayResponse and verifies the supplier's signature.
-//
-// - Returns the RelayResponse, even if basic validation fails (may contain error reason).
-// - Verifies supplier's signature with the provided publicKeyFetcher.
-func ValidateRelayResponse(
```

Comment:
IMO we should bring this back and figure out how to refactor this so it's possible.

It made more sense before.

Reply:
Ditto. See PR #39